doc: fixed building command for VEGAboard

howardliu7874 opened the pull request:
Signed-off-by: Howard Liu <howardliu7874@hotmail.com>

dbkinder reviewed (approved):
doc changes LGTM.  @mbolivar you agree?

mbolivar commented on boards/riscv32/rv32m1_vega/doc/index.rst:
```diff
@@ -573,7 +573,7 @@ Windows (run this in a ``cmd`` prompt, from the Zephyr directory)::
   # Use CMake to generate a Ninja-based build system:
   type NUL > empty.ld
-  cmake -GNinja -DBOARD=rv32m1_vega_ri5cy -DCMAKE_REQUIRED_FLAGS=-Wl,-dT=empty.ld ..
+  cmake -GNinja -DBOARD=rv32m1_vega_ri5cy -DCMAKE_REQUIRED_FLAGS=-Wl,-dT=../../empty.ld ..
```
Hm, if it's a path issue, why not `%cd%\empty.ld` or so?

howardliu7874 replied:
Good point. I'll use it. thanks.

howardliu7874 committed:
doc: fix build command on Windows for VEGAboard

The path was changed during generating Ninja-based build system,
so the path of link script file was wrong when testing the
cross-compiler. Fix it by giving the absolute path for link script.

Signed-off-by: Howard Liu <howardliu7874@hotmail.com>

mbolivar reviewed (approved):
Thanks for the fix! LGTM

MaureenHelm reviewed (approved):
Thanks @howardliu7874 !
